extras/userdoc/md/documentation/simulations-with-gap-junctions.md

```diff
@@ -0,0 +1,121 @@
+# Simulations with gap junctions
+
+**Note:** This documentation describes the usage of gap junctions in NEST 2.12.
+A documentation for NEST 2.10 can be found in
+[Hahne et al. 2016](http://link.springer.com/chapter/10.1007/978-3-319-50862-7_4).
+It is however recommended to use NEST 2.12 (or higher), 
+due to several improvements in terms of usability. 
+
+## Introduction
+
+Simulations with gap junctions are supported by the Hodgin-Huxley neuron model `hh_psc_alpha_gap`.
+The synapse model to create a gap-junction connection is named `gap_junction`. 
+Unlike chemical synapses gap junctions are bidirectional connections. 
+In order to create **one** accurate gap-junction connection 
+**two** NEST connections are required:
+
+```python
+import nest
+
+a = nest.Create('hh_psc_alpha_gap')
+b = nest.Create('hh_psc_alpha_gap')
+# Create gap junction between neurons a and b
+nest.Connect(a, b, {'rule': 'one_to_one'}, {'model': 'gap_junction', 'weight': 0.5})
+nest.Connect(b, a, {'rule': 'one_to_one'}, {'model': 'gap_junction', 'weight': 0.5})
+```
+
```

Comment:
It might be dangerous to start with code lines which are actually not working, in particular if the user quickly wants to look up the call and does not read the whole documentation. I would suggest to skip the lines 16 to 28 and directly start with the correct function call. If you think these lines are necessary you could put them as a **note** after the correct call (around line 50) and say something like "these lines would in principle lead to the same gap junction connection but are forbidden in NEST 2.12 to prevent the user from creating incomplete gap junctions and thus result in an error."